modules/ztp-spoke-factory-pipeline.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="edge-factory-pipeline_{context}"]
+= The edge factory pipeline
+
+This stage deploys and configures the edge clusters. When this pipeline is completed, the edge clusters are ready for use when the enclosure gets relocated to the end customer's remote site.
+
+The flow associated with deploying the edge cluster is:
+
+Check hub::
+
+This step installs the various tools on the edge cluster that are needed. It downloads `jq`, `oc`, `opm` and `kubectl`. It proceeds to verify that various hub install prerequisites exist before proceeding, for example it checks the:
+
+* {product-title} version.
+* Nodes are ready.
+* Cluster Operators are ready.
+* Metal^3^ pods are ready.
+* Persistent volumes are created.
+* DNS requirements are satisfied.
+
+Deploy edge::
+
+This step starts with the edge cluster provisioning. This process ends with pushing a notification from the edge cluster to the hub and answering with an ACK.
+
+Deploy NMState and MetalLB::
+
+This step deploys NMState and the MetalLB operators that creates one profile per node to obtain an IP from the external network's DHCP. Then the MetalLB creates a resource called an AddressPool to build the relationship between the internal and external interface using a LoadBalancer interface. Finally it creates a service for the API and the ingress. Without this step you will not be able to access the API or ingress by using the external address.
+
+Deploy OpenShift Data Foundation::
+
+This step deploys the Local Storage Operator and also OpenShift Data Foundation (ODF). The Local Storage Operator uses the node disks (NVMEs) to create persistent volumes. ODF uses the NVMEs to deploy the StorageCluster on top of them. ODF generates the storage classes and dynamically provisions the persistent volumes. This provides the storage necessary to host the disconnected registry images (Quay).
+
+Deploy Quay::
+
+This step deploys the Quay Operator and components of Quay, because the final customer needs a fully supported solution in the edge and the factory is expected to have their own internal registry. This Quay deployment has a small footprint enabling only the features needed to host an internal registry with basic functions.
+
+Deploy workers::
+
+This step deploys the worker nodes, and adds these to the edge cluster.
+
+Deploy UI::
+
+This step runs in parallel with the deploy workers and the detach cluster stages. The deploy UI stage helps to simplify the configuration of the cluster when relocated.
```

Comment:
The deploy UI stage helps to simplify the configuration of the edge cluster after it's relocated to the customer's site.